_data/objectfeatures.json

```diff
@@ -1389,5 +1389,5 @@
    }
   ]
  },
- "ilastik.__version__": "1.4.0rc6"
+ "ilastik.__version__": "1.4.1b16.dev1+ged23825fb.d20240319"
```

Comment:
Oops?

I guess this isn't really relevant for anything though

Reply:
well it's the version of the code I ran it on...

Reply:
I can rerun once the ilastik PR is merged, then the reference should point to something more reachable (but still dev-y)

Reply:
Sure, though I don't think it's an issue anyway